src/diskio/mod.rs

```diff
@@ -187,6 +310,51 @@ pub fn write_file<P: AsRef<Path>, C: AsRef<[u8]>>(
     Ok(())
 }
 
+#[allow(unused_variables)]
```

Comment:
Why do we need this lint?

Reply:
Windows vs Linux. mode isn't used on Windows.

Reply:
:+1: